lib/Controller.js

```diff
@@ -1,7 +1,8 @@
 /**
  * Created by elad.benedict on 8/26/2015.
  */
-
+// parse command line args and update config obj
+var regressionConfig = require('./Adapters/RegressionAdapter/RegressionConfig.js').getConfig();
```

Comment:
moved to AdapterFactory.js. Controller code cleaned from unit tests related code.